### PR TITLE
Add Prometheus egg

### PR DESCRIPTION
**Pull request by pascal-zarrad**
### Description
This PR adds a Prometheus egg that allows to run Prometheus in Pterodactyl.
The egg is based on the debian base image.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [X] Does your submission pass tests (server is connectable)?
2. [X] Does your server use a custom docker image? - _it doesn't_
3. [X] Have you added the server to the main README.md?
4. [X] Have you added a unique README.md for the server you are adding?

### Additional context
This is the companion of #1227 and allows to host Grafana and Prometheus in Pterodactyl.

### CODE REVIEW

File: database/tsdb/prometheus/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Prometheus
+
+### From the [Prometheus](https://www.mongodb.com/) GitHub repository
```
Comment (parkervcp) [inline database/tsdb/prometheus/README.md:3]: This is not MongoDB...

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -62,6 +62,9 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [MariaDB](/database/sql/mariadb)
 * [PostgreSQL](/database/sql/postgres)
 
+### Timeseries Databases
+* [Prometheus](/database/tsdb/prometheus)
```
Comment (parkervcp) [inline README.md:66]: Prometheus is a monitoring tool

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: database/tsdb/prometheus/README.md (hunk 1 above)
Comment (pascal-zarrad) [reply database/tsdb/prometheus/README.md:3]: Whoops, missed that one

File: README.md (hunk 2 above)
Comment (pascal-zarrad) [reply README.md:66]: Well, Prometheus is marketed more or less as being a "Monitoring system & time series database". But I will move it to an `monitoring` directory as it fits the purpose of Prometheus way better (and its actually the use-case I use Prometheus for). 

A `monitoring `directory does not really fit into the database folder neither into the `software` directory.
Where would you suggest to place that directory?

File: database/tsdb/prometheus/README.md (hunk 1 above)
Comment (pascal-zarrad) [reply database/tsdb/prometheus/README.md]: Fixed

File: README.md (hunk 2 above)
Comment (parkervcp) [reply README.md:66]: make a monitoring folder if anything.

Comment (pascal-zarrad) [reply README.md]: Prometheus has been moved to a dedicated `monitoring` directoryy. The main README has also been updated.

**Review by Software-Noob — APPROVED** [review]
LGTM

**Review by parkervcp — APPROVED** [review]

**Pull request merged**